pkg/translator/jaeger/metadata.yaml

```diff
@@ -0,0 +1,3 @@
+status:
+  codeowners:
+    active: [open-telemetry/collector-approvers]
```

Comment:
Here too. Perhaps @frzifus could own this?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS#L130C1-L131C1
```
pkg/translator/jaeger/                                   @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
```

Sure, if @frzifus is open to the idea, please open an issue.

Reply:
hm.. y, I can take it - But be prepared for some pings in the future.